_sass/_team.scss

```diff
@@ -0,0 +1,25 @@
+.team {
+	&_member {
+		padding: 2rem 0;
+		border-bottom: 1px solid #efefef;
+	}
+	&_image {
+		border-radius: 50%;
+		border: solid 1px $primary-color;
+		margin: 0 0 auto 0;
+	}
+	&_information {
+		display: inline-block;
+		margin: 0 0 0 1rem;
+		vertical-align: top;
+	}
+	&_description {
+		color: #888;
+	}
+	&_twitter {
+		color: #1b95e0;
+		&:hover {
+			color: #0c7abf;
+		}
+	}
+}
```

Comment:
BEM should be used to describe layout (how it looks) and not content (team member).